api/office/schema.py

```diff
@@ -120,15 +144,20 @@ def resolve_offices(self, info, **kwargs):
 
 
 class Query(graphene.ObjectType):
+    """
+        Query the ofiices
```

Comment:
great work @TeamoreA kindly work on this typo. `ofiices` to `offices`

Reply:
Okay, am on it